### PR TITLE
[13.0][MIG] partner_risk_insurance: Migration to 13.0

### PR DESCRIPTION
**Pull request by Tardo**
cc @tecnativa TT21636

@sergio-teruel @carlosdauden

### CODE REVIEW

**Review by acsonefho — APPROVED** [review]

**Review by ThomasBinsfeld — APPROVED** [review]

File: partner_risk_insurance/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "Factor Libre S.L,"
+    "NaN·tic,"
+    "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "website": "https://github.com/OCA/credit-control",
+    "depends": ["base"],
+    "category": "Credit Control",
+    "data": ["security/ir.model.access.csv", "views/res_partner_view.xml"],
+    "installable": True,
```
Comment (sergio-teruel) [inline partner_risk_insurance/__manifest__.py:18]: Add maintainers key

File: partner_risk_insurance/tests/test_partner_risk_insurance.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.tests import SavepointCase
+
+
+class TestSaleOrderLineInput(SavepointCase):
+    at_install = False
```
Comment (sergio-teruel) [inline partner_risk_insurance/tests/test_partner_risk_insurance.py]: Change this by @tagged('post_install', '-at_install')

**Review by sergio-teruel — APPROVED** [review]
little comment

**Comment by Tardo** [comment]
Changes done!

**Comment by Tardo** [comment]
Ready to merge

**Pull request merged**